perf/perftool.py

```diff
@@ -51,7 +60,19 @@ def setUp(self):
         self.srcdir = os.path.join(self.srcdir, 'perftool-testsuite-master')
 
     def test(self):
+        '''
+        perf test :Does sanity tests
+        Execute the tests by calling each module
+        '''
         self.count = 0
+        # Built in perf test
+        for string in process.run("perf test").stderr.splitlines():
+            if 'FAILED' in str(string.splitlines()):
+                self.count += 1
+                self.log.info("Test case failed is %s"
```

Comment:
i think you forgot to address previous review  comment here , self.fail() is appropriate here